ee/tabby-webserver/src/repositories/resolve.rs

```diff
@@ -1,88 +1,98 @@
-use std::{
-    collections::HashMap,
-    ops::Deref,
-    path::PathBuf,
-    str::FromStr,
-    sync::{Arc, RwLock},
-};
+use std::{path::PathBuf, str::FromStr, sync::Arc};
 
-use anyhow::Result;
+use anyhow::{anyhow, Result};
 use axum::{
     body::boxed,
     http::{header, Request, Uri},
     response::{IntoResponse, Response},
     Json,
 };
 use hyper::Body;
+use kv::{Bucket, Config, Store};
 use serde::{Deserialize, Serialize};
 use tabby_common::{config::RepositoryConfig, SourceFile, Tag};
 use tower::ServiceExt;
 use tower_http::services::ServeDir;
-use tracing::{debug, error, warn};
 
-use crate::{
-    cron::{CronEvents, StartListener},
-    schema::repository::RepositoryService,
-};
+use crate::schema::repository::RepositoryService;
+
+fn repository_meta_db() -> PathBuf {
+    tabby_common::path::tabby_root().join("repositories.kv")
+}
```

Comment:
https://github.com/TabbyML/tabby/blob/837e36ded9a36ce9e02e8afa466f99fbda04a54c/ee/tabby-db/src/path.rs#L5

refactor this into `webserver/src/path.rs`. pass db_file into `tabby_db::DbConn::new` call